.gitignore

```
@@ -1,2 +1,5 @@
 # Notebooks 
-*.ipynb_checkpoints*
\ No newline at end of file
+*.ipynb_checkpoints*
+
+# Environments
+env*/
```

I was just about to add this in `main` directly, but maybe just include it here so that you don't have to fix the conflit?
```suggestion
env*/

*.fits
```

---

Works for me!